rubicon_ml/intake_rubicon/publish.py

```diff
@@ -1,9 +1,16 @@
+from typing import TYPE_CHECKING, Optional
+
 import fsspec
 import yaml
 
+if TYPE_CHECKING:
+    from rubicon_ml.viz.experiments_table import ExperimentsTable
+
 
 def publish(
     experiments,
+    # visualization object passed, defaulted to None
```

Comment:
I appreciate this comment, but types are self documenting and this is not needed going forward.